TOKENS.md

```diff
@@ -2,7 +2,7 @@
 
 The `cosign` command line tool optionally supports hardware tokens for signing and key management.
 This support is enabled through the [PIV protocol](https://csrc.nist.gov/projects/piv/piv-standards-and-supporting-documentation)
-and the [go-piv](https://github.com/go-piv/piv-go) library, which is not included in the standard release. Use [`make cosign-pivkey`](https://github.com/sigstore/cosign/blob/a8d1cc1132d4a019a62ff515b9375c8c5b98a5c5/Makefile#L52), or `go build -tags=pivkey`, to build `cosign` with support for hardware tokens.
+and the [go-piv](https://github.com/go-piv/piv-go) library, which is not included in the standard release. Use `make cosign-pivkey-pkcs11key`, or `cd cmd/cosign; go build -tags=pivkey`, to build `cosign` with support for hardware tokens.
```

Comment:
instead of the `cd cmd/cosign` would use

```
go build -tags=pivkey,pkcs11key -o cosign ./cmd/cosign
```

Reply:
Very nice. Done, though I left out the `-o cosign` which seems optional. Make sense?